src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -319,10 +291,18 @@ public function doChangePassword(array $data, $form)
      *
      * @return HTTPResponse
      */
-    public function redirectBackToForm()
+    public function redirectBackToForm(?int $withMemberID = null, ?string $withToken = null)
```

Comment:
We can't amend public method signatures in a minor release, as they count as part of our "Public API" -  see https://docs.silverstripe.org/en/5/project_governance/public_api/